jetty-proxy/src/test/java/org/eclipse/jetty/proxy/AsyncMiddleManServletTest.java

```diff
@@ -625,14 +630,19 @@ public void transform(ByteBuffer input, boolean finished, List<ByteBuffer> outpu
                     }
                 };
             }
-        });
+        };
+        startServer(serverServlet);
+        startProxy(proxyServlet);
```

Comment:
Restore inlining of `AsyncMiddleManServlet`.

Reply:
Restored